tests/test-common.js

```diff
@@ -42,7 +42,7 @@ const TEST_TYPES = {
   },
   'jld:ExpandTest': {
     skip: {
-      specVersion: ['json-ld-1.1']
+      regex: [/#t[cmn]/, /#t008./]
```

Comment:
Not sure how best to handle this, but I had to change it to `/#t008[0-7]/` so it ran 0088 and the new tests from https://github.com/json-ld/json-ld.org/pull/567.

Reply:
That certainly will do the job.